src/pymor/algorithms/timestepping.py

```diff
@@ -69,7 +70,8 @@ def solve(self, initial_time, end_time, initial_data, operator, rhs=None, mass=N
         operator
             The |Operator| A.
         rhs
-            The right-hand side F (either |VectorArray| of length 1 or |Operator| with
+            The right-hand side F (either |VectorArray| of length 1 or |VectorArray| of length
+            equal to the number of time-steps to perform or |Operator| with
```

Comment:
Is it ok, to make this claim about the number of array elements here in `TimeStepper`? Would a higher order methods maybe expect the number of array elements to be the number of time steps times the number of stages? (For adaptive time stepping, we'd probably want a continuous-in-time interpolation?)